lib/ansible/modules/network/junos/junos_ping.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Ansible by Red Hat, inc
```

Comment:
Use 2019 instead and use the latest license template https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/cli/cli_config.py#L1-L5

Reply:
Fixed.